package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "eigengraus-essential-establishment-generator",
+  "version": "0.1.0",
+  "description": "Eigengrau's Essential Establishment Generator",
+  "main": "main.txt",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator.git"
+  },
+  "author": "ryceg",
+  "license": "GNU GENERAL PUBLIC LICENSE",
+  "bugs": {
+    "url": "https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/issues"
+  },
+  "homepage": "https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/blob/master/README.md",
+  "devDependencies": {
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb-base": "^13.1.0",
```

Comment:
Should probably be `eslint-config-standard`

Reply:
Has been updated